### ISSUE TITLE
Hello, has the original code been deleted？

### ISSUE THREAD

**Issue by ghost** [BartyzalRadek/Multi-label-Inception-net#23]
Hello, has the original code been deleted,

**Comment by BartyzalRadek** [comment]
No? What do you mean?

**Comment by BartyzalRadek** [comment]
Oh right the https://codelabs.developers.google.com/codelabs/tensorflow-for-poets 
Yeah it appears so, however this codebase is now obsolete since it does not supprt TF 2.0.
Since it's a simple edit of the original script for retraining the Inception Net, you can find some newer implementation based on Tf2 nad probably Keras (that would simplify a lot of the code) and edit that according to the Medium article: 
https://towardsdatascience.com/multi-label-image-classification-with-inception-net-cbb2ee538e30
